packages/teams-js/src/internal/utils.ts

```diff
@@ -402,16 +402,3 @@ export function inServerSideRenderingEnvironment(): boolean {
 }
 
 const appIdRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
```

Comment:
Is this still used somewhere?

Reply:
Nope. So I removed it. Thanks for pointing it out.